docs/make.jl

```diff
@@ -21,6 +21,11 @@ include("generate.jl")
 include("changelog.jl")
 create_documenter_changelog()
 
+bibtex_plugin = CitationBibliography(
+    joinpath(@__DIR__, "src", "assets", "references.bib"),
+    style=:authoryear
```

Comment:
I prefer the default `:numeric` style I think.

Reply:
The reasoning from my side was that people might be able to deduce some information from the name already when reading. Changed.